toqito/matrix_ops/commutant.py

```diff
@@ -5,7 +5,7 @@
 
 
 def commutant(A: np.ndarray | list[np.ndarray]) -> list[np.ndarray]:
-    r"""Compute an orthonormal basis for the commutant algebra :cite:`qetlab_commutant`.
+    r"""Compute an orthonormal basis for the commutant algebra :cite:`johnston2016qetlab`.
```

Comment:
The notion of a commutant does not come from QETLAB though. The reference implementation, yes, but citing QETLAB after defining the commutant I think is misleading.

Reply:
It's context dependent. If you describe *what* the commutant is, then you shouldn't cite QETLAB. The definition of the commutant does not derive from QETLAB. You should instead cite some source that describes and defines *what* the commutant is. However, we should *also* cite QETLAB but elsewhere to inform the user that the implementation here is *derived* from the implementation from QETLAB. Does that make sense / seem reasonable to you? @fs1132429

Reply:
@vprusso yep! thanks a lot!

Reply:
> You should instead cite some source that describes and defines what the commutant is. 

@fs1132429 Vincent's suggestion is missing.